runtime/src/main/java/io/quarkus/logging/sentry/SentryConfig.java

```diff
@@ -22,6 +22,12 @@ public class SentryConfig {
     @ConfigItem(name = ConfigItem.PARENT)
     boolean enable;
 
+    /**
+     * Determine whether to enable the Sentry logging extension.
+     */
+    @ConfigItem
+    public Optional<Boolean> enabled;
```

Comment:
FWIW I think it makes sense to avoid this `~` YAML trick. It causes all sorts of issues.

Ideally, I think it should be `active` though if we want to follow the convention we use in the core. `enabled` is for enabling/disabling the extension at build time. `active` is for activating/deactivating the behavior at runtime. I think it's more the latter in this case, right?

@ia3andy WDYT?

If we include this one, could you mark the old one as deprecated so that the doc is not confusing?

Reply:
@rmanibus could you rename to active and add deprecated to the other config?